drivers/usb/device/usb_dc_numaker.c

```diff
@@ -0,0 +1,1901 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_numaker_usbd
+
+#include <soc.h>
+#include <zephyr/kernel.h>
+#include <zephyr/usb/usb_device.h>
+#include <zephyr/dt-bindings/usb/usb.h>
+#include <zephyr/sys/math_extras.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/clock_control_numaker.h>
+#include <zephyr/drivers/reset.h>
+#include <zephyr/drivers/pinctrl.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(usb_dc_numaker, CONFIG_USB_DRIVER_LOG_LEVEL);
+
+#include "NuMicro.h"
```

Comment:
It's better as:
#include <NuMicro.h>

More, #include <soc.h> & #include <NuMicro.h> platform specific header files should locate after standard/general ones .

Reply:
fixed as suggested